modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+= Use Flow Designer to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Flow Designer is a tool in Anypoint Design Center that enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To use Flow Designer to configure a connector:
+
+. Configure the input source (trigger) for the flow.
+. Configure the target connector operation.
+
+[[configure-input-source]]
+== Configure the Input Source for the Flow
+
+. Log into Anypoint Platform.
+. Click *Start designing*.
+. In Design Center, click *Create new*.
+. Click *Create new application*.
+. Enter a project name and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
++
+.Clicking *Go straight to canvas* takes you to a trigger card.
+image::introduction/intro-config-fd-go-to-canvas.png[Exit from the Let's Get Started screen]
++
+. Click on the empty trigger card in the middle of the canvas to expand it.
+. Configure one of these input sources for the connector:
+* <<http-listener,HTTP Listener>>
+* <<scheduler,Scheduler>>
+* <<connector-specific,A connector-specific input source>>
+
+[[http-listener]]
+=== Use HTTP Listener as an Input Source
+
+`HTTP Listener` listens for HTTP requests on a specified path and initiates the associated flow when it receives a request. To use `HTTP Listener` as an input source:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Search* field, enter *HTTP*.
+. In the *Operation* field, click *HTTP Listener*.
+. In the *Path* field, specify the path on which `HTTP Listener` listens for requests.
+. Click *Edit*:
```

Comment:
When would a user just use the default CloudHub HTTP configuration versus adding a connection and using that in a custom configuration? Why not use the connection that is in the default CloudHub HTTP configuration?